dev/com.ibm.ws.microprofile.config.1.4/src/com/ibm/ws/microprofile/config14/impl/Config14ProviderResolverImpl.java

```diff
@@ -20,4 +22,9 @@
     protected AbstractConfigBuilder newBuilder(ClassLoader classLoader) {
         return new Config14BuilderImpl(classLoader, getScheduledExecutorService(), getInternalConfigSources());
     }
+
+    @Override
+    protected void setStaticInstance(ConfigProviderResolver instance) {
+        //NO-OP ... this version relies purely on the ServiceLoader mechanism
+    }
```

Comment:
This is picky, but I don't like having a method that says that it does something, and then is overridden to not do that thing.

I don't think it's important for this prototype, but I think I'd prefer a more vague name for the method.